clientgui/ViewWork.cpp

```diff
@@ -261,7 +276,8 @@ CViewWork::CViewWork(wxNotebook* pNotebook) :
     m_aStdColNameOrder->Insert(_("Progress"), COLUMN_PROGRESS);
     m_aStdColNameOrder->Insert(_("Status"), COLUMN_STATUS);
     m_aStdColNameOrder->Insert(_("Elapsed"), COLUMN_CPUTIME);
-    m_aStdColNameOrder->Insert(_("Remaining (estimated)"), COLUMN_TOCOMPLETION);
+    m_aStdColNameOrder->Insert(_("Remaining"), COLUMN_TOCOMPLETION);
```

Comment:
I added the "(estimated)" part of the column name back.

Reply:
To your other point, I'm making a change so that the Estimated Completion column shows "---" if the Elapsed column is also "---" (i.e. elapsed CPU time is <= 0).